x-pack/plugins/global_search_providers/public/providers/get_app_results.ts

```diff
@@ -62,6 +85,13 @@ export const scoreApp = (term: string, appLink: AppLink): number => {
   return 0;
 };
 
+const scoreAppByKeywords = (term: string, keywords: string[]): number => {
+  const scores = keywords.map((keyword) => {
+    return scoreAppByTerms(term, keyword);
```

Comment:
optional: we could break/return earlier if a keyword's score is 100.

Reply:
We can add that as an optional optimization strategy if/when plugin owners start adding their keywords. Right now, I don't think it's needed.